app-crypt/argon2/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>simon-bgo@slevermann.de</email>
+		<name>Simon Levermann</name>
+	</maintainer>
+	<longdescription>Argon2 is a password hashing tool and library that won the Password Hashing Competition (PHC)</longdescription>
```

Comment:
You must add the proxy maintainer group to the metadata file. See https://gitweb.gentoo.org/repo/gentoo.git/tree/app-editors/atom/metadata.xml for instance.